tools/bin/hawqregister

```diff
@@ -729,6 +728,23 @@ class HawqRegister(object):
                 for k, eof in enumerate(eofs[1:]):
                     query += ',(%d, %d, %d, %d, %d)' % (self.firstsegno + k + 1, eof, -1, -1, -1)
```

Comment:
Yes, upgrade to 3.x is a big deal - I'm definitely *not* proposing that. Rather, we should minimize the technical debt by making net-new code (like this tool) better aligned with 3.x conventions, to avoid revisiting it later on.